app/services/analytics_events.rb

```diff
@@ -1969,7 +1969,7 @@ def idv_doc_auth_submitted_image_upload_form(
   # @option extra [Boolean] 'OrientationChanged'
   # @option extra [Boolean] 'PresentationChanged'
   # @param ["Passport","DriversLicense"] document_type Document capture user flow
-  # The document capture image was uploaded to vendor during the IDV process
```

Comment:
issue: Please make sure to keep the general description of the event! (Our data analytics and fraudops folks use our docs to build reports and do investigations)

Reply:
Thank you; good catch.